src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const thumbs = document.getElementById('thumbs');
+const largeImg = document.getElementById('largeImg');
+
+thumbs.onclick = (image) => {
```

Comment:
```suggestion
thumbs.onclick = (event) => {
```
It's not an image, but event